Validation/README.md

```diff
@@ -44,39 +32,35 @@ Using Validation without a virtual environment and outside of the container is n
 
 
 ## Usage
-This module _cannot_ be run from ldmx-sw directory.
-`import Validation` prefers the local directory instead of the installed path so it tries to
-load from the `ldmx-sw/Validation` directory.
-
-We could fix this by renaming the package inside Validation or renaming the directory.
 
 ### CLI
 The Validation module is constructed to do some common tasks quickly on the command line.
 Printing out its help message shows how to run it and gives you the details on what
 parameters to provide.
 ```
-python3 -m Validation -h
+cd Validation/src
+denv python3 -m Validate -h
```

Comment:
This brings up a large usage question. The whole point of installing the python package is to allow users to run `denv python3 -m Validate` from _anywhere_. In this example, you are particularly running from within the source code for the python module which isn't bad, it just makes the install step not necessary.

There are two ways forward:
1. Remove the install step: we don't need a `install-validation` recipe and instead we have a `validation-plot` recipe which does the necessary directory movement and passes the arguments to the module.
2. Keep the install step and make sure it can be run from anywhere. The current recipe does install the `Validate` python and I can get the help message from other places besides within the source directory.

I like the install step since then folks can run `denv python3 -m Validate` from where-ever their data happens to reside, but I also don't use the Validation module much and so if you prefer running the Validation source directly (maybe because you are editing it frequently?), then we could move in that direction. I just think we should only have one way of running the code to avoid confusion.

Reply:
Yeah I'm good with the solution that allows us to run this from anywhere, I'll modify the README accordingly

Reply:
Done now!